src/Util/IngestUtil.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Util;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+use GuzzleHttp\Exception\GuzzleException;
+use GuzzleHttp\Exception\ServerException;
+
+use App\Exception\HttpClientErrorException;
+use App\Exception\HttpServerErrorException;
+
+/**
+ * A utility query the Ingest server API.
+ */
+class IngestUtil
+{
+    /**
+     * The url for the API.
+     *
+     * @var string
+     */
+    private $url;
+
+    /**
+     * Constructor.
+     *
+     * @param string $ingestApiUrl The API URL.
+     */
+    public function __construct(
+        string $ingestApiUrl
+    ) {
+        $this->url = $ingestApiUrl;
```

Comment:
url is too generic, please name is insgestApiUrl as well. This is consistent with have we have done this in other places.